api/services/ocean/spark/schemas/oceanSparkApplication.yaml

```diff
@@ -0,0 +1,160 @@
+type: object
+required:
+- internalId
+- id
+- displayName
+- accountId
+- organizationId
+- userId
+- clusterId
+- controllerClusterId
+- appState
+- submissionSource
+- createdAt
+- updatedAt
+allOf:
+- $ref: "../../../../commons/schemas/createdAtUpdatedAt.yaml"
+- type: object
+  description: >
+    The description of a Spark application
+  properties:
+    internalId:
+      $ref: "internalId.yaml"
+    id:
+      $ref: "oceanSparkApplicationId.yaml"
+    displayName:
+      $ref: "oceanSparkApplicationDisplayName.yaml"
+    accountId:
+      $ref: "accountId.yaml"
+    organizationId:
+      $ref: "organizationId.yaml"
```

Comment:
@HHK1 @arnarpall those fields are clear from the context, so we could remove them from the API.
I think you noted this down already, but I just wanted to confirm.
I wonder if it's clear from the context in all usages of the data class though.

Reply:
Yeah absolutely. The `userId` `organizationId` and `accountId` are all context fields and we should just remove them from the API

Reply:
Not the `userId` no?

Reply:
Yes we should keep the `userId`, it is not clear from the context

Reply:
I will remove those two fields from the doc, will be implemented in https://spotinst.atlassian.net/browse/BGD-1669